### PR TITLE
chore: upgrade AzureDisk/AzureFile CSI Drivers to v0.9.0

### PR DESCRIPTION
**Pull request by ZeroMagic**
/cc @andyzhangx

### CODE REVIEW

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Comment by ZeroMagic** [comment]
@xuto2 Can I get an approval?

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -543,7 +543,7 @@ for ADDON_IMAGE in ${ADDON_IMAGES}; do
 done
 
 AZUREDISK_CSI_VERSIONS="
-0.7.0
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:546]: if 0.7.0  is being used by exsiting cluster shall we keep it until 0.9.0 is fully in effect (as part of the ccp chart upgrade )?

Comment (ZeroMagic) [reply vhdbuilder/packer/install-dependencies.sh:546]: Yes, we should keep 0.7.0 version. Updated PR.

**Pull request merged**